pages/index.js

```diff
@@ -36,12 +37,23 @@ function Home({ data, preview }) {
     return null
   }
 
-  const { sections, title } = pageData
+  const { sections, title, seo } = pageData
 
   return (
     <>
       <Head>
-        <title>{title}</title>
+        <title>{seo?.seoTitle || title}</title>
+        {/* Primary Meta Tags */}
+        <meta name="title" content={seo?.seoTitle} />
+        <meta name="keywords" content={seo?.seoKeywords} />
+        <meta name="synonyms" content={seo?.seoSynonyms} />
+        <meta name="description" content={seo?.seoDescription} />
+        <meta name="image" content={urlFor(seo?.seoImage)} />
+        {/* Open Graph / Facebook */}
+        <meta name="og:type" content="website" />
+        <meta name="og:title" content={seo?.seoTitle} />
+        <meta name="og:description" content={seo?.seoDescription} />
+        <meta name="og:image" content={urlFor(seo?.seoImage)} />
       </Head>
```

Comment:
I realized we should create a SEO component for this one so that we could use this in the `index` page and in `preview` page. We do this to prevent any inconsistencies that might be introduced when we edit two files.

Usage:

```
<SEO data={pageData} />
```

Reply:
Created an SEO component